cloud/aws/templates/aws_oidc/main.tf

```diff
@@ -22,7 +28,8 @@ resource "aws_db_instance" "civiform" {
     Name = "${var.app_prefix} Civiform Database"
     Type = "Civiform Database"
   }
-
+  availability_zone               = data.aws_availability_zones.available.names
```

Comment:
The name name suggests that this param takes a single zone. But `available.names` returns a list. Is that ok?

Reply:
you're correct, it can only be one and its picked automatically, so we don't need to set it.